src/view/com/util/text/RichText.tsx

```diff
@@ -51,7 +51,7 @@ export function RichText({
       <Text
         testID={testID}
         type={type}
-        style={[style, pal.text, lineHeightStyle]}
+        style={[style, pal.text, lineHeightStyle, s.flex1]}
```

Comment:
We can't just put flex: 1 on the component itself because that's a layout behavior that may not be desired in all cases. It has to be passed in as a style on the component when it's used other components.

Reply:
What do you think about passing the flex1 as a style from the outside?

only the colored parts have flex: 1 now.

**Feed ITem:**

<img width="1037" alt="image" src="https://github.com/bluesky-social/social-app/assets/2494766/6d236798-b0ba-4c2f-ae64-4eca808e4717">

**Thread:** 

<img width="1262" alt="image" src="https://github.com/bluesky-social/social-app/assets/2494766/e7c5f45d-c880-4c37-ac2e-4f1555e50cdc">

**Profile:**

<img width="1026" alt="image" src="https://github.com/bluesky-social/social-app/assets/2494766/481f16d8-6379-4b70-837f-fc5e823aa521">